web/gds-user-ui/src/modules/dashboard/member/index.tsx

```diff
@@ -14,6 +15,15 @@ const MemberPage: React.FC = () => {
       <Heading marginBottom="69px">
         <Trans>TRISA Member Directory</Trans>
       </Heading>
+      <Card maxW="100%" marginBottom={6}>
+        <Card.Body>
+          <Text>
+            <Trans>
+              The TRISA Member Directory is for informational purposes and is meant to foster collaboration between members. It is available to verified TRISA contacts only. If you cannot view the list, please complete the registration process. You can view the member list after verification is complete.
+            </Trans>
+          </Text>
+        </Card.Body>
+      </Card>
```

Comment:
Would you be interested in creating a MemberNotification component instead and applying React's memoization ?

Reply:
Sure, I'll make that change before merging.